app/views/how-tos/switching-from-govuk-prototype-kit.html

```diff
@@ -0,0 +1,105 @@
+{% extends 'layout.html' %}
+
+{% block pageTitle %}
+  Switching from the GOV.UK prototype kit - NHS.UK prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="nhsuk-grid-row">
+
+    <div class="nhsuk-grid-column-two-thirds">
+
+      <h1>
+        Switching from the GOV.UK prototype kit
+      </h1>
+
+
+<p>
+      If you have learned how to use the GOV.UK prototype kit, you will be able to use the NHS.UK prototype kit. This is because the two prototype kits are built in the same way. However, you must do some things differently.
+    </p>
+<h2>Installing and running the NHS.UK prototype kit</h2>
+
+<p>
+When you are installing and running your prototype:
+
+{{ list({
+  title: "Do",
+  type: "tick",
+  items: [
+    {
+      item: 'check when you install your prototype that you are using the right version of node.js - you may need to install and use a different version. If you need to make prototypes that for both NHS.UK and GOV.UK you can switch versions of node with a <a href="https://www.freecodecamp.org/news/node-version-manager-nvm-install-guide/">node version manager</a>'
```

Comment:
Both kits require Node version 20 at the moment, so this may be less of an issue (unless you have an older version of either kit)?

Might still be worth mentioning but perhaps not the thing to start with?

Node.js is also capitalised I think.

```suggestion
      item: 'check when you install your prototype that you are using the right version of Node.js - you may need to install and use a different version. If you need to make prototypes that for both NHS.UK and GOV.UK you can switch versions of node with a <a href="https://github.com/nvm-sh/nvm">node version manager</a>'
```